index.yaml

```diff
@@ -39,10 +39,10 @@ entries:
         - https://github.com/apache/incubator-yunikorn-release/releases/download/v0.8.0/yunikorn-0.8.0.tgz
       version: 0.8.0
     - apiVersion: v1
-      appVersion: 0.9.0
-      created: "2020-06-24T14:12:46.578917+02:00"
+      appVersion: latest
```

Comment:
is it OK to set appVersion to latest instead of a numeric version?

Reply:
The app version is an optional field and it is the version of the application the chart contains, in our case the YK version